Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift

```diff
@@ -363,6 +316,72 @@ extension MerchantHeadlessCheckoutAvailablePaymentMethodsViewController: PrimerH
     }
 }
 
+extension MerchantHeadlessCheckoutAvailablePaymentMethodsViewController {
+    
+    private func setupSessionLogic() {
+        clientSession = MerchantMockDataManager.getClientSession(sessionType: klarnaSession ? .klarna : .generic)
```

Comment:
We have an issue here - all settings changed on the previous screen are overriden by this mocked data.

Reply:
I changed they approach, the switch is now called `One time payment` since this is what it does, adds the necessary data to the clientSession in order to perform a successful `ONE_OFF_PAYMENT`.
I refactored the MerchantMockDataManager to reflect these changes.
And now the switch only populates the fields with the data, it doesn't change the clientSession anymore, so the user changed fields values will not be impacted after the user goes to Available Payment Methods controller.